GitUpKit/Views/GIMapViewController.h

```diff
@@ -37,6 +37,7 @@
 @property(nonatomic) BOOL forceShowAllTips;
 - (BOOL)selectCommit:(GCCommit*)commit;  // Also scrolls if needed to ensure commit is visible - Returns YES if commit was selected
 - (GINode*)nodeForCommit:(GCCommit*)commit;
+- (NSPoint)positionForCommit:(GCHistoryCommit*)historyCommit;
```

Comment:
This should take a `GCCommit` for consistency with the other APIs and I think the name should reflect better what's the coordinate system e.g. `-positionInViewForCommit:`.